test/smartseq2_single_sample/pr/TestSmartSeq2SingleCellPR.wdl

```diff
@@ -0,0 +1,59 @@
+import "ss2_single_sample.wdl" as target
+import "ValidateSmartSeq2SingleCell.wdl" as checker
+
+
+# this task will be run by the jenkins script that gets executed on our PRs.
+
+workflow TestSmartSeq2SingleCellPR {
+
+  # output hashes
+  String expected_counts_hash
+  String expected_alignment_summary_metrics_hash
+
+  # SS2 inputs
+   # load annotation
+  File gtf_file
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+  #load index
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+  # ref index name
+  File hisat2_ref_name
+  File hisat2_ref_trans_name
+  # samples
+  String stranded
+  String sample_name
+  String output_name
+  File fastq1
+  File fastq2
+
+  call target.SmartSeq2SingleCell as target {
```

Comment:
I would be a little weary of aliasing your call with the same alias as your import if nothing else just for readability

Reply:
Interesting, we do this a lot. How would you fix this? Would `target_wdl` for the import and `target_workflow` for the execution be a good solution?